evap/evaluation/models.py

```diff
@@ -617,6 +617,18 @@ def can_results_page_be_seen_by(self, user):
             return self.is_user_responsible_or_contributor_or_delegate(user)
         return self.can_be_seen_by(user)
 
+    @property
+    def can_reset_to_new(self):
+        allowed_sources = [
+            Evaluation.State.PREPARED,
+            Evaluation.State.EDITOR_APPROVED,
+            Evaluation.State.APPROVED,
+            Evaluation.State.IN_EVALUATION,
+            Evaluation.State.EVALUATED,
+            Evaluation.State.REVIEWED,
+        ]
+        return self.state in allowed_sources and not self.is_single_result
```

Comment:
Any reason to prefer this over the following?
```suggestion
        return Evaluation.State.NEW < self.state < Evaluation.State.PUBLISHED and not self.is_single_result
```